sigs.yaml

```
@@ -2710,7 +2710,7 @@ workinggroups:
     time: "11:00"
     tz: PT (Pacific Time)
     frequency: biweekly (On demand)
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```

```suggestion
    url: https://zoom.us/j/4799874685
```
From agenda: https://docs.google.com/document/d/1j3vrG6BgE0hUDs2e-1ZUegKN4W4Adb1B6oJ6j-4kyPU